src/rpc/net.cpp

```diff
@@ -556,7 +556,8 @@ static UniValue getnetworkinfo(const JSONRPCRequest& request)
 static UniValue setban(const JSONRPCRequest& request)
 {
     const RPCHelpMan help{"setban",
-                "\nAttempts to add or remove an IP/Subnet from the banned list.\n",
+                "\nAttempts to add or remove an IP/Subnet from the banned list.\n"
+                "Peers that are automatically banned cannot be unbanned.\n",
```

Comment:
This line should be removed (sorry - this was from my branch when I was referring to automatic banning)

Reply:
Done.